spec/unit/plugin/pagination_spec.rb

```diff
@@ -53,6 +53,16 @@
     end
   end
 
+  describe '#total_pages' do
+    it 'returns the exact number of pages to accommodate all elements' do
+      users = rom.relation(:users).page(1).per_page(3)
+      expect(users.pager.total_pages).to eql(3)
+      
+      users = rom.relation(:users).per_page(9)
+      expect(users.pager.total_pages).to eql(1)
+    end
```

Comment:
If you want, I can separate them, but I think they should be together. They are testing the same property: that `total_pages` only adds +1 when the number of elements divided by the page size is a whole number (no decimals)